examples/blend_modes.rs

```diff
@@ -58,65 +60,73 @@ impl MainState {
 
         // draw the diagram
         for i in 0..3 {
-            self.circle.draw(
-                ctx,
-                DrawParam::default()
+            canvas.draw_mesh(
+                self.circle.clone(),
+                None,
+                graphics::DrawParam::new()
                     .dest(pos + Vec2::from(REL_POSITIONS[i]))
                     .color(TRI_COLORS[i]),
-            )?;
+            );
         }
 
         // draw text naming the blend mode
-        let text = graphics::Text::new((name, self.font, 20.0));
+        canvas.set_blend_mode(BlendMode::ALPHA);
```

Comment:
This currently doesn't have any kind of effect. The text still gets drawn with whatever blend mode was set before.

Reply:
I'm not sure what you mean exactly, but the blend_modes example should now produce an identical result to the original renderer.

Reply:
My mistake, [0bc5f18](https://github.com/ggez/ggez/pull/1020/commits/0bc5f18a7cf0081e708b5c49692a6e3db7315897) fixed it.